web/eslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```

Comment:
A month isn't that long :) When I was doing it for bi it was different. I'll change